pandas/core/window.py

```diff
@@ -780,8 +780,41 @@ def calc(x):
 
 class _Rolling_and_Expanding(_Rolling):
 
-    _shared_docs['count'] = """%(name)s count of number of non-NaN
-    observations inside provided window."""
+    _shared_docs['count'] = dedent(r"""
+    The %(name)s sum if it is a non-Nan value inside the window.
+
+    Returns
+    -------
+    Returns the object type with the sum of the window values
```

Comment:
Can you make note of what the return type is? Also similar comment as above - referring to `sum` within a `count` function is confusing. I'm not sure `current row` is the right concept here, though I understand what you are trying to get at

Reply:
You can use a similar wording as here: https://github.com/pandas-dev/pandas/pull/20263/files

Reply:
That makes sense.  How does this work?

Returns
    -------
    Series or Dataframe
        Returned object type is determined by the caller of %(name)

Reply:
I would just copy verbatim, so `Returned object type is determined by the caller of the %(name)s calculation`